xwiki-platform-core/xwiki-platform-xclass/xwiki-platform-xclass-test/xwiki-platform-xclass-test-docker/src/test/it/org/xwiki/xclass/test/ui/AllIT.java

```diff
@@ -33,31 +34,37 @@
 
 /**
  * Tests the default class sheet (XWiki.ClassSheet).
- * 
+ *
  * @version $Id$
- * @since 4.5
+ * @since 12.3
  */
-public class ClassSheetTest extends AbstractTest
+@UITest
+public class AllIT
```

Comment:
That's doesn't look good. If you're moving to docker tests, the AllIT should only contains call to nested tests according to our practice. And this actual class should be named something like ClassSheetIT.

Reply:
Indeed, looks like I'd missed this step.
What's the rational behind this modularization ?

Reply:
Readability when there’s more than 1 test. You don’t need an AllIT when the module has a single test BTW!

Reply:
> What's the rational behind this modularization ?

the idea was to have one scenario by class and to execute them as Nested JUnit 5 tests. The rationale if I remember properly is that we only execute AllIT.java (as you specified in the pom) which prevents building several times the configurations for docker tests.

Reply:
Ok, thanks. I'll do the refactoring to introduce a ClassSheetIT class.